datahub-web-react/src/App.tsx

```diff
@@ -52,21 +64,38 @@ const App: React.VFC = () => {
         register.register(new TagEntity());
         return register;
     }, []);
+
+    const theme: Theme = useMemo(() => {
+        const overridesWithoutPrefix: { [key: string]: any } = {};
+        const themeConfig = dynamicThemeConfig || defaultThemeConfig;
+        Object.assign(overridesWithoutPrefix, themeConfig.styles);
+        Object.keys(overridesWithoutPrefix).forEach((key) => {
+            overridesWithoutPrefix[key.substring(1)] = overridesWithoutPrefix[key];
```

Comment:
Why do we remove the "@" here... what if people don't use the @ when they specify styles?